teradatalabs/mapr52-hive/files/socks-proxy.sh

```diff
@@ -0,0 +1 @@
+../../../commons/socks-proxy.sh
```

Comment:
@sanjay990  new line  is not required as per artur , its same in other containers also